test-configs.yaml

```diff
@@ -7,6 +7,7 @@ file_system_types:
   buildroot:
     url: 'https://storage.kernelci.org/images/rootfs/buildroot/kci-2018.11'
     arch_map:
+      arc:     [{arch: arc}]
```

Comment:
While this should work, it's not actually necessary to add an entry if the arch name is the same for both the kernel and the distro conventions.